source/georef/georef.cpp

```diff
@@ -417,9 +417,10 @@ void GeoRef::build_autocomplete_list(){
             // Same way for all address in the admin.
             // After this modification the result found with postal code in search string
             // should contain only this postal code but not others of the admin found.
-            std::string key = way->way_type + " " + way->name + " " + admin->name + " " + admin->postal_codes_to_string();
-            fl_way.add_string(key, pos, this->ghostwords, this->synonyms);
-
+            if (way->visible == true) {
```

Comment:
minor: you should do a continue with the check on the `name` and `edges`

Reply:
It already exist ?
https://github.com/CanalTP/navitia/pull/2592/files#diff-992196e00edd443955062e4bad390541R409

Reply:
The point of @kinnou02 is to put it the same way up there I guess:
```cpp
if (way->visible == false) { continue; }
```